modules/json/pom.xml

```diff
@@ -33,6 +33,18 @@
       <artifactId>spring-data-commons</artifactId>
       <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.module</groupId>
+      <artifactId>jackson-module-parameter-names</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.datatype</groupId>
+      <artifactId>jackson-datatype-jdk8</artifactId>
```

Comment:
this one is IMHO also not required for this PR. It IMHO adds support for optinals.
Maybe we should leave it optional if someone wants support for optionals. For our modules we should keep dependencies as minimal as possible.

Reply:
@vapadwal 👍 Thanks for updating this. Done:
https://github.com/vapadwal/devon4j/blob/issue_javatime8/modules/json/pom.xml#L38